src/nord.sh

```diff
@@ -266,13 +266,14 @@ print_help() {
 # @return 0 if all required dependencies are validated, 1 otherwise
 # @since 0.2.0
 validate_dependencies() {
-  declare -a missing_deps deps=("${!1}")
+  declare -a deps=("${!1}")
+  declare -a missing_deps=()
```

Comment:
Let's assign the value separately which is safer for Bash.

```suggestion
  declare -a missing_deps deps
  deps=("${!1}")
  missing_deps=()
```